core/src/main/java/com/radixdlt/environment/EventProcessorOnDispatch.java

```diff
@@ -64,33 +64,25 @@
 
 package com.radixdlt.environment;
 
-import com.radixdlt.consensus.event.CoreEvent;
-import java.util.Objects;
+import com.radixdlt.consensus.event.LocalEvent;
 import java.util.Optional;
 
 /**
- * A processor which is to process an event synchronously at time of dispatch
+ * A processor which is to process an event synchronously at time of dispatch. Note that this
+ * processor can be used for both types of events, local and remote.
```

Comment:
this comment needs an update; this processor is for local events only

Reply:
Fixed too.